packages/document/docs/en/api/commands.mdx

```diff
@@ -35,9 +35,10 @@ Options:
 The `rspress preview` command is used to preview the output files of the `rspress build` command locally.
 
 ```txt
-Usage: rspress preview [options]
+Usage: rspress preview [root] [options]
```

Comment:
Can we consider aligning with Rsbuild? `-r --root <root>`

https://rsbuild.dev/guide/basic/cli#rsbuild-preview

Reply:
rspress dev / build use this style of root option.

Reply:
get